lib/util/git/index.ts

```diff
@@ -862,11 +864,13 @@ export async function commitFiles(
   config: CommitFilesConfig
 ): Promise<CommitSha | null> {
   const commitResult = await prepareCommit(config);
-  if (!commitResult) {
-    return null;
+  if (commitResult) {
+    const pushResult = await pushCommit(config);
+    if (pushResult) {
+      return fetchCommit(config);
+    }
```

Comment:
Isn't as linear, but doesn't break coverage.
I'll try to come up with unit test in further PR.